src/frontend/src/helpers/Storage/LocalStorage.ts

```diff
@@ -56,9 +59,10 @@ class LocalStorage implements AbstractStorage {
     * @param {string} file_history: The history of the file.
     * @param {string || any false value} checksum : The online checksum of the file, false to not update (offline write).
     */
-  public async writeFile(fid: FileID, contents?: string): Promise<void> {
```

Comment:
Ok this might take some time. I'll keep this on my list. Can I worry about it later since you're in a hurry to go?